modules/nf-core/btyper3/meta.yml

```
@@ -0,0 +1,49 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "btyper3"
+description: Taxonomic classification of Bacillus cereus group isolates from assemblies
+keywords:
+  - bacteria
+  - classify
+  - fasta
+tools:
+  - "btyper3":
+      description: "In silico taxonomic classification of Bacillus cereus group isolates using assembled genomes"
+      homepage: "https://github.com/lmc297/BTyper3"
+      documentation: "https://github.com/lmc297/BTyper3"
+      tool_dev_url: "https://github.com/lmc297/BTyper3"
+      doi: "10.3389/fmicb.2020.580691"
+      licence: ["GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
```

```suggestion
        e.g. `[ id:'test' ]`
```
Probably does not apply here?